stdci.yaml

```diff
@@ -1,3 +1,7 @@
+archs: x86_64
```

Comment:
Why do you change stdci.yaml and check-merged? in Travis PR?

Reply:
my bad make a checkout from another PR and not from master.